keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -755,7 +763,7 @@ type ExemptPriorityLevelConfiguration struct {
   //
   // Bigger numbers mean a larger nominal concurrency limit,
   // at the expense of every other Limited priority level.
-  // This field has a default value of 30.
+  // This field has a default value of 1.
```

Comment:
I don't think we can change the default without revving the api, it will make config behave differently. (Imagine provisioning a new cluster with yaml manifests that rely on the default value)

Reply:
Oh wait, this is not shared with the other options and is a new field, right? This is fine then.

Reply:
This is not a change.  See, this is in a new type?

Reply:
@wojtek-t : brain fart.  Yes, I think we can make the default zero.

See also the remark, currently at new lines 790--794, about hoisting the common fields into the containing struct.  That will eliminate the need for `ExemptPriorityLevelConfiguration` and make the default value for the hoisted `NominalConcurrencyShares` field depend on the `Type PriorityLevelEnablement` field (0 in one case, 30 in the other).

Reply:
> Yes, I think we can make the default zero.

So I would vote for making it 0.

> See also the remark, currently at new lines 790--794, a

I'm aware of it, but we will not do that for existing API types - we can do that as part of promotion to v1.
And it's kind-of refactoring, we should preserve the default once we do that.
That's why I would like to use 0 here.

Reply:
I do not think the choice of default here matters to what happens in the refactoring; the refactoring can preserve whatever default we want.  My point above was that doing so involves making the refactored defaulting logic have to look a the `Type PriorityLevelEnablement` field.  The only thing that would obviate that dependency would to make the default here match the default for non-exempt (30), which I think we are agreed we do not want.